src/components/Wallet/Asset.tsx

```diff
@@ -1,26 +1,135 @@
+'use client';
 import Button from '@/atoms/Button';
-import React from 'react';
+import React, { useState } from 'react';
 import { Horizon } from 'stellar-sdk';
+import Modal from '../Modal';
+import useBoolean from '@/hooks/useBoolean';
+import Form from '../Form';
+import Input from '@/atoms/Input';
+import { errorMsg, optionsAsync, succesMsgAsync } from '@/utils/toastMsg';
+import { MessageError } from '@/utils/constants';
+import { parserAmountToDecimal } from '@/utils/parserAmount';
+import { VALIDATIONS } from '@/utils/validations';
+import { useBearStore } from '@/store/store';
+import { sendTransaction } from '@/services/payment';
+import { toast } from 'react-toastify';
+import LoaderAndText from '@/molecules/LoaderAndText';
+import useLoadAccount from '@/hooks/useLoadAccount';
 interface Props {
   balance:
     | Horizon.BalanceLineNative
     | Horizon.BalanceLineAsset<'credit_alphanum4'>
     | Horizon.BalanceLineAsset<'credit_alphanum12'>
     | Horizon.BalanceLineLiquidityPool;
 }
+
+interface Transaction {
+  publicKey: string;
+  amount: string;
+}
+interface State {
+  transaction: Transaction;
+}
+const INITIAL_STATE: State['transaction'] = {
+  amount: '',
+  publicKey: '',
+};
 export default function Asset({ balance }: Props) {
+  const [{ amount, publicKey }, setTransaction] =
+    useState<State['transaction']>(INITIAL_STATE);
+  const { view, handleChangeBoolean } = useBoolean();
+  const { getData } = useLoadAccount();
```

Comment:
I think we talked about avoiding the data word, everything's data. Make it more descriptive, what type of data are you getting?